examples/05-Q3D/Q2D_Armoured_Cable.py

```diff
@@ -0,0 +1,359 @@
+mod2D.set_working_coordinate_system('CS_c_strand_1')
+c1_id = mod2D.create_circle(['0mm','0mm','0mm'],'c_strand_radius',name='c_strand_1', matname='copper')
+c2_id = c1_id.duplicate_along_line(vector=['0mm','2.0*c_strand_radius','0mm'],nclones=2)
+mod2D.duplicate_around_axis(c2_id,cs_axis="Z",angle = 360/core_n_strands,nclones=6)
+c_unite_name = mod2D.unite(Q2D_DS.get_all_conductors_names())
+
+fill_id = mod2D.create_circle(['0mm','0mm','0mm'],'3*c_strand_radius',name='c_strand_fill', matname='plastic_pp_carbon_fiber')
+fill_id.color=(255,255,0)
+xlpe_id = mod2D.create_circle(['0mm','0mm','0mm'],'3*c_strand_radius+'+str(core_xlpe_ins_thickness)+'mm',name='c_strand_xlpe',
+					matname='plastic_pe_cable_grade')
+xlpe_id.color=(0,128,128)
+
+mod2D.set_working_coordinate_system('Global')
+all_obj_names = Q2D_DS.get_all_conductors_names()+Q2D_DS.get_all_dielectrics_names()
+mod2D.duplicate_around_axis(all_obj_names,cs_axis="Z",angle = 360/cable_n_cores,nclones=4)
+cond_names = Q2D_DS.get_all_conductors_names()
+
+#####################################################################################
+# Create geometry for filling object
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+filling_id = mod2D.create_circle(['0mm','0mm','0mm'],'filling_radius',name='Filling', matname='plastic_pp_carbon_fiber')
+filling_id.color=(255,255,180)
+#####################################################################################
+# Create geometry for inner sheat object
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+inner_sheat_id = mod2D.create_circle(['0mm','0mm','0mm'],'inner_sheat_radius',name='Inner_Sheat', matname='PVC plastic')
+inner_sheat_id .color=(0,0,0)
+
+#####################################################################################
+# Create geometry for armature fill
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+arm_fill_id = mod2D.create_circle(['0mm','0mm','0mm'],'armour_radius',name='ArmourFilling', matname='plastic_pp_carbon_fiber')
+arm_fill_id.color=(255,255,255)
+
+#####################################################################################
+# Create geometry for outer sheat
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+outer_sheat_id = mod2D.create_circle(['0mm','0mm','0mm'],'outersheat_radius',name='OuterSheat', matname='PVC plastic')
+outer_sheat_id.color= (0,0,0)
+
+#####################################################################################
+# Create geometry for armature steel strands
+# ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+arm_strand_1_id = mod2D.create_circle(['0mm','armour_centre_pos','0mm'],'1.1mm',name='arm_strand_1', matname='steel_stainless')
+arm_strand_1_id.color = (128,128,64)
+arm_strand_1_id.duplicate_around_axis('Z','360deg/n_arm_strands',nclones='n_arm_strands')
+arm_strand_names = mod2D.get_objects_w_string('arm_strand')
+
+#####################################################################################
+# Create Region
+# ~~~~~~~~~~~~~
+oeditor = Q2D_DS.odesign.SetActiveEditor("3D Modeler")
```

Comment:
```suggestion
Q2D_DS.modeler.create_region([500, 500, 500, 500, 0, 0])
```
@tizianrot you could achieve the region creation with the pyaedt method.